backend/src/apiserver/config/config.json

```diff
@@ -11,6 +11,10 @@
     "BucketName": "mlpipeline",
     "PipelinePath": "pipelines"
   },
+  "ArchiveConfig": {
+    "LogFileName": "main.log",
+    "LogPathPrefix": "/artifacts"
+  },
```

Comment:
We are moving towards env var friendly config names, so I'd suggest the following:
```
  "ARCHIVE_CONFIG": {
    "LOG_FILE_NAME": "main.log",
    "LOG_PATH_PREFIX": "/artifacts"
  },
```

and when overriding the config from env var, we can use `ARCHIVE_CONFIG_LOG_FILE_NAME`

Reply:
We have env var friendly names now